transport/wseb/src/main/java/org/kaazing/gateway/transport/wseb/WsebConnectProcessor.java

```diff
@@ -47,23 +47,19 @@
 import org.kaazing.gateway.transport.http.HttpMethod;
 import org.kaazing.gateway.transport.http.HttpProtocol;
 import org.kaazing.gateway.transport.http.HttpSession;
-import org.kaazing.gateway.transport.ws.WsBinaryMessage;
 import org.kaazing.gateway.transport.ws.WsCommandMessage;
 import org.kaazing.gateway.transport.ws.WsMessage;
-import org.kaazing.gateway.transport.ws.WsPingMessage;
-import org.kaazing.gateway.transport.ws.WsPongMessage;
-import org.kaazing.gateway.transport.ws.bridge.filter.WsBuffer;
 import org.kaazing.gateway.transport.wseb.filter.WsebFrameCodecFilter;
-import org.kaazing.mina.core.buffer.IoBufferAllocatorEx;
 import org.kaazing.mina.core.buffer.IoBufferEx;
 import org.kaazing.mina.core.future.DefaultWriteFutureEx;
 import org.kaazing.mina.core.future.WriteFutureEx;
+import org.kaazing.mina.core.session.IoSessionEx;
 import org.kaazing.mina.core.write.DefaultWriteRequestEx;
 import org.kaazing.mina.core.write.WriteRequestEx;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class WsebConnectProcessor extends BridgeConnectProcessor<WsebSession> {
+class WsebConnectProcessor extends BridgeConnectProcessor<WsebSession> {
```

Comment:
Not sure what you mean. I would have liked to make it BridgeConnectProcess<TransportSession> but that's not possible without introducing a whole new transport layer which I am trying to avoid (see also my previous comment about why WsebAcceptProcessor cannot directly be the processor on TransportSession)